node/packages/aws-lambda-sdk/trace-spans/aws-lambda.js

```diff
@@ -14,9 +14,10 @@ const arch = (() => {
     case 'arm64':
       return 'arm64';
     default:
-      process.stderr.write(
-        `Serverless SDK Warning: Unrecognized architecture: "${process.arch}"\n`
-      );
+      console.warn({
+        source: 'serverlessSdk',
+        message: `Unrecognized architecture: "${process.arch}"\n`,
```

Comment:
And maybe this one under `arch`?

Reply:
This one is actually controversial, as I don't think it can ever happen. Either environment would have to change so our detection will stop working, or we will have to use e.g. arm arch build in some new arch as introduced by AWS

Reply:
Yeah true as far as I know I don't think there is a way to provide a custom arch for lambda functions